src/api/verificationRequestApi.ts

```diff
@@ -13,6 +14,33 @@ const request = axios.create({
     baseURL: `/api/verification-request`,
 });
 
+const createVerificationRequest = (
+    t,
+    router,
+    verificationRequest: any = {}
+) => {
+    const { nameSpace = NameSpaceEnum.Main } = verificationRequest;
```

Comment:
chore (non-blocking): create a tech debt task to make this namespace API client interface more modular.